test/APIJSON/non-swift-api.swift

```diff
@@ -21,7 +21,7 @@ public func callNative ()
       my_native_c()      
 }
 
-// CHECK:        "target": "x86_64-apple-macosx10.9",
+// CHECK:        "target": 
```

Comment:
may I suggest to use 
`"target": "{{.*}}-apple-macosx10.9"` 
or 
`"target": "{{(arm64)|(x86_64)}}-apple-macosx10.9"`
options?

to not to lose `apple-macosx10.9` part check

Reply:
> may I suggest to use
> `"target": "{{.*}}-apple-macosx10.9"`
> or
> `"target": "{{(arm64)|(x86_64)}}-apple-macosx10.9"`
> options?
> 
> to not to lose `apple-macosx10.9` part check

The reason why `apple-macos` part works is because the requirement for the test, which probably should not hard enforced here.

The two real options are:
1. Hard code arch for both `swift-frontend` for building the module and the check
2. Just ignore the targets.

I just pick the second one, since the target is not the important part of the test. The test is to check all the interfaces are detected from the module.

Reply:
got it. thx for explanation!